Mathlib/Algebra/Group/Equiv/Basic.lean

```diff
@@ -163,6 +165,14 @@ def postcompEquiv {α β : Type*} [Monoid α] [Monoid β] (e : α ≃* β) (γ :
   left_inv _ := by ext; simp
   right_inv _ := by ext; simp
 
+/-- The monoid isomorphism `(M →* N₁) ≃* (M →* N₂)` obtained by postcomposition with
+a multiplicative equivalence `e : N₁ ≃* N₂`. -/
+@[to_additive "The monoid isomorphism `(M →+ N₁) ≃+ (M →+ N₂)` obtained by postcomposition with
+an additive equivalence `e : N₁ ≃+ N₂`."]
+def postcompMulEquiv (e : N₁ ≃* N₂) : (M →* N₁) ≃* (M →* N₂) where
```

Comment:
For linear maps, this is called `LinearEquiv.congrRight`

Reply:
Oh no, my new definition is actually a special case of the existing `MulEquiv.monoidHomCongr`. What do we do?

Reply:
My take on these things in the past has been that we should have separate left and right congrs, rather than one big congr, to avoid having to write lemmas like `bigCongr x y = (bigCongr x id).comp (bigCongr id y)`.

My guess would be that the LinearMap api is most mature, and so we should copy that in the absence of a strong opinion.